appeals/e2e/cypress/e2e/back-office-appeals/scheduleSiteVisit.spec.js

```diff
@@ -7,19 +7,19 @@ import { UpdateDueDatePage } from '../../page_objects/updateDueDatePage';
 const appealsListPage = new AppealsListPage();
 const updateDueDatePage = new UpdateDueDatePage();
 
-describe.skip('Appeals feature', () => {
+describe('Appeals feature', () => {
 	beforeEach(() => {
 		cy.login(users.appeals.caseAdmin);
 	});
 
 	it('Change to acommpanied site visit from case timetable', () => {
-		cy.visit('/appeals-service/appeals-list');
-		appealsListPage.clickAppealFromList(20);
+		cy.visit('/appeals-service/all-cases');
+		appealsListPage.clickAppealFromList(30);
 		appealsListPage.clickChangeVisitTypeHasCaseTimetable();
 		appealsListPage.selectRadioButtonByValue('Accompanied');
 		updateDueDatePage.enterVisitDay('29');
```

Comment:
Doesn't this have to be a future date, will the tests stop working in 29/12/2025? Should it be a dynamic date?